src/pages/index.js

```diff
@@ -2,51 +2,90 @@ import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import popupWithConfirmation from "../components/popupWithConfirmation.js";
 import UserInfo from "../components/UserInfo.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
 import {
-  initialCards,
   settings,
   galleryDisplay,
   nameInput,
   titleInput,
   imgUrlInput,
+  initialCards,
   descriptionInput,
   imgModalForm,
   editButton,
   addImgButton,
   profileForm,
   nameOfUser,
   jobOfUser,
+  avatarOfUser,
+  deleteButton,
+  deleteForm,
+  avatarForm,
+  avatarUpdateButton,
 } from "../utils/constants.js";
 import "../pages/index.css";
 
+let userId;
+let section;
 // Creating a new instances of Classes-----------
 const cardFormValidator = new FormValidator(settings, imgModalForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```